src/chttpd_auth_cache.erl

```diff
@@ -209,3 +206,28 @@ update_doc_ignoring_conflict(DbName, Doc, Options) ->
         throw:conflict ->
             ok
     end.
+
+maybe_validate_user_creds(nil) ->
```

Comment:
a `-spec` here would help readability, so we know what the possible results are supposed to be.